examples/tutorial/jupyter/execution/omnisci_on_native/local/exercise_1.ipynb

```diff
@@ -0,0 +1,233 @@
+   "source": [
+    "# Note: Do not change this code!\n",
+    "import numpy as np\n",
+    "import pandas\n",
+    "import sys\n",
+    "import modin\n",
+    "import modin.config as cfg\n",
+    "cfg.StorageFormat.put('omnisci')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pandas.__version__"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "modin.__version__"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Implement your answer here. You are also free to play with the size\n",
+    "# and shape of the DataFrame, but beware of exceeding your memory!\n",
+    "\n",
+    "import pandas as pd\n",
+    "\n",
+    "frame_data = np.random.randint(0, 100, size=(2**10, 2**5))\n",
+    "df = pd.DataFrame(frame_data)\n",
+    "\n",
+    "# ***** Do not change the code below! It verifies that \n",
+    "# ***** the exercise has been done correctly. *****\n",
+    "\n",
+    "try:\n",
+    "    assert df is not None\n",
+    "    assert frame_data is not None\n",
+    "    assert isinstance(frame_data, np.ndarray)\n",
+    "except:\n",
+    "    raise AssertionError(\"Don't change too much of the original code!\")\n",
+    "assert \"modin.pandas\" in sys.modules, \"Not quite correct. Remember the single line of code change (See above)\"\n",
+    "\n",
+    "import modin.pandas\n",
+    "assert pd == modin.pandas, \"Remember the single line of code change (See above)\"\n",
+    "assert hasattr(df, \"_query_compiler\"), \"Make sure that `df` is a modin.pandas DataFrame.\"\n",
+    "\n",
+    "print(\"Success! You only need to change one line of code!\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that we have created a toy example for playing around with the DataFrame, let's print it out in different ways."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Concept for Exercise: Data Interaction and Printing\n",
+    "\n",
+    "When interacting with data, it is very imporant to look at different parts of the data (e.g. `df.head()`). Here we will show that you can print the modin.pandas DataFrame in the same ways you would pandas."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Print the first 10 lines.\n",
+    "df.head(10)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Print the DataFrame.\n",
+    "df"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Free cell for custom interaction (Play around here!)\n",
+    "df.add_prefix(\"col\").count()"
```

Comment:
This "play around" suggestion will definitely result in some problems if somebody would actually try to do so. The `df` has integer columns which will result in an error almost on any operation due to #3370. We bypass this error for a single operation here by adding a string prefix, however, it's not obvious at all that it must be applied for every single function:
```python
# Free cell for custom interaction (Play around here!)
df.add_prefix("col").count() # Everything works
df.sum() # User tries its own code and everything crashes
```

My suggestion is to do `df = df.add_prefix("col")` with the appropriate comment of why we need it at the first cell, right after the `df` is constructed.

Reply:
> My suggestion is to do df = df.add_prefix("col") at the first cell right after the df is constructed, with the appropriate comment of why we need it.

makes sense

Reply:
@YarShev @dchigarev I changed this.